plugins/logger-json/src/templates/logger.config.ts

```diff
@@ -0,0 +1,16 @@
+import { ConfigService } from "@nestjs/config";
+import { Params } from "nestjs-pino"
+
+ADDITIONAL_LOG_PROPERTIES_KEY
```

Comment:
@sakurablossoms01 this file won't compile as `ADDITIONAL_LOG_PROPERTIES_KEY` statement is not a ts valid statement.

Did you mean to somehow get this value from somewhere? I could not see other references in the generated code.